### ISSUE TITLE
Payload-type consistency enforced across endpoints?

### ISSUE THREAD

**Issue by markreg** [jitsi/jitsi-videobridge#133]
Should a specific payload-type be enforced for the conference and all associated endpoints since JVB doesn't transcode?

**Comment by bgrozev** [comment]
We don't want to explicitly make the assumption that payload-type mappings are the same for all endpoints, and try to work without it whenever possible, but in our environment (jitsi-meet + jicofo) it holds. So problems might creep in if you don't enforce the same PT mappings.

**Comment by gpolitis** [comment]
ditto what @bgrozev said, we unfortunately have a few constants here and there that assume 116 for RED, 100 for VP8.

**Comment by joqn** [comment]
JVB *does* support transcoding in mixer mode.  Feel free to put in more checks for the translator mode, but please don't break the transcoding in mixer mode.

**Comment by markreg** [comment]
Right, since JVB is first and foremost an SFU, clients must be vetted for codec compatibility before endpoint creation as far as I understand. e.g. Chrome/Firefox SDP would need to be checked
